history/peter-toma.md

```diff
@@ -0,0 +1,11 @@
+---
+description: Programmer who developed the oldest commercial machine translation program
+---
+# Peter Toma
+Peter Toma was a computer programmer who designed Systran, the oldest commercial machine translation program. 
+
+He initially developed the program for the United States Air Force. 
+It was intended to translate large-scale scientific documents from Russian into English during the Cold War. 
+
+It was later acquired by the European Commission for English-to-French direction. 
+The language pairs got expanded when the program was purchased and popularized by a French company. 
```

Comment:
Let's just make an article about Systran, and this kind of thing can go there.

We can say that he was at Caltech.  Was he at Georgetown too?